oblt-cli/cluster-create-custom/action.yml

```diff
@@ -0,0 +1,103 @@
+    description: 'Suffix to be appended to the randomised cluster name'
+    required: false
+  slack-channel:
+    description: 'The slack channel to notify the status.'
+    default: '#observablt-bots'
+    required: false
+  username:
+    description: 'Username to show in the deployments with oblt-cli, format: [a-z0-9]'
+    default: 'apmmachine'
+    required: false
+  gitops:
+    description: 'Whether to provide the GitOps metadata to the oblt-cli'
+    default: false
+    required: false
+  dry-run:
+    description: 'Whether to dryRun'
+    default: false
+    required: false
+  skip-random-name:
+    description: 'Whether to deploy a cluster with a random name'
+    default: false
+    required: false
+runs:
+  using: "composite"
+  steps:
+    - name: Configure oblt-cli flags for gitops - issues
+      id: gitops
+      if: ${{ !github.event.issue.pull_request && contains(inputs.gitops, 'true') }}
+      run: |
+        {
+          echo -n '{'
+          echo -n '"GitOps":"true",'
+          echo -n '"GitHubRepository":"${{ github.repository }}"'
+          [ -n '${{ github.event.issue.number }}' ] && echo -n ',"GitHubIssue":"${{ github.event.issue.number }}"'
+          echo -n '}'
+        } > .gitops
+        echo "GITOPS=$(cat .gitops)" >> $GITHUB_OUTPUT
+        rm .gitops
+      shell: bash
+
+    - name: Configure oblt-cli flags for gitops - pull-request
+      id: gitops-pr
+      if: github.event.issue.pull_request && contains(inputs.gitops, 'true')
+      run: |
+        {
+          echo -n '{'
+          echo -n '"GitOps":"true",'
+          echo -n '"GitHubRepository":"${{ github.repository }}",'
+          echo -n '"GitHubCommit":"${{ github.event.comment.id }}",'
+          echo -n '"GitHubPullRequest":"${{ github.event.issue.number }}"'
+          echo -n '}'
+        } > .gitops
+        echo "GITOPS=$(cat .gitops)" >> $GITHUB_OUTPUT
+        rm .gitops
+      shell: bash
+
+    - name: Configure oblt-cli flags
+      id: flags
+      run: |
+        {
+          echo -n '--template=${{ inputs.template }} '
+          [ -n '${{ inputs.cluster-name-prefix }}' ] \
+            && echo -n '--cluster-name-prefix=${{ inputs.cluster-name-prefix }} '
+          [ -n '${{ inputs.cluster-name-suffix }}' ] \
+            && echo -n '--cluster-name-suffix=${{ inputs.cluster-name-suffix }} '
+          [ '${{ inputs.dry-run }}' == 'true' ] \
+            && echo -n '--dry-run '
+          [ '${{ inputs.skip-random-name }}' == 'true' ] \
+            && echo -n '--skip-random-name '
+        } > .flags
+        echo "COMMAND=$(cat .flags)" >> $GITHUB_OUTPUT
+        rm .flags
+        # Merge GITOPS with the PARAMETERS
+        MERGED=$(echo '${{ inputs.parameters }} ${{ steps.gitops.outputs.GITOPS || steps.gitops-pr.outputs.GITOPS }} {"EphemeralCluster":"true"}' | jq -c -s add)
+        echo "PARAMETERS=${MERGED}" >> $GITHUB_OUTPUT
+      shell: bash
+
+    - uses: elastic/oblt-actions/oblt-cli/setup@v1.2.0
```

Comment:
What's the strategy here about the versioning?

Reply:
It's just copied from the serverless action.

IMO, what we put here doesn't really matter. The only requirement is that the action exists on the particular release.

It will be updated by Dependabot (as soon as it's configured)